### ISSUE TITLE
[FEATURE] Additional columns in multitiles

### ISSUE THREAD

**Issue by pilpfiction** [mennovanhout/flutter-hdashboards-issues#111]
**Is your feature request related to a problem? Please describe.**
Feature request.

**Describe the solution you'd like**
The multitiles are great. While I can add many devices only one capability can be shown. I would like to be a able to select and display additional capabilities, which basically results in a table of values. 

Example: show the current and target temperature of thermostates in one single multitile.

**Describe alternatives you've considered**
Currently I have to create to separate multitiles, like this: 
![Screenshot_20231202-115714](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/4cc8b486-fc94-4317-ba51-6e6737ec4076)

**Additional context**
None

**Comment by StaffordBully** [comment]
you can add the same device twice
first add the device Himmel target
second the device Himmel current

That how i do it

**Comment by MP-ICT-Diensten** [comment]
+1

**Comment by jeesnl** [comment]
For each sensor you could also create a multi tile with target a temp above each other. 
I expect multi row and multi column together becomes to complex.

**Comment by pilpfiction** [comment]
> For each sensor you could also create a multi tile with target a temp above each other. I expect multi row and multi column together becomes to complex.

Yes also an alternative. But this is than similar to adding the device (thermostates) and not selecting a capability (standard use case). Then I get temp and target temp displayed in one tile. 
While using a multitile for one device and adding two capabilities, brings then no advantage to the standard use case in my view.

**Comment by jeesnl** [comment]
Make one multi card for each device with a header then it’s good to read.

Op za 2 dec 2023 om 13:32 schreef pilpfiction ***@***.***>

**Comment by pilpfiction** [comment]
This doesn't really bring an advantage at all as opposed to display the device (thermostate) with no capability. In this case the temp and target temp are displayed:
![Screenshot_20231202-134916](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/221c3749-96bd-4993-9197-84ace2d43904)

**Comment by StaffordBully** [comment]
something like this would be great
![image](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/144690805/40fb05c4-32b1-4930-b2f0-78c37fa80500)

**Comment by pilpfiction** [comment]
> something like this would be great ![image](https://private-user-images.githubusercontent.com/144690805/287448849-40fb05c4-32b1-4930-b2f0-78c37fa80500.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1MjI1NDMsIm5iZiI6MTcwMTUyMjI0MywicGF0aCI6Ii8xNDQ2OTA4MDUvMjg3NDQ4ODQ5LTQwZmIwNWM0LTMyYjEtNDkzMC1iMmYwLTc4YzM3ZmE4MDUwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwMlQxMzA0MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yN2E3ODlkYjI3ZGEwNTI5M2JjMDIyNDQzZTIzZGI5MjY4Y2JkZDU0OWUwOWIxZmNlNjI2YjI1Njk3OGU3MWE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wY9mibVi8fF9cAo29OlgW-gqRRFw4Mc21IWgmSu-4Yg)

Exactly.

This is how I currently do it: I am using two multitiles with all devices and one selected capability next to each other.

But as you can see it uses a lot of space. The second icon and title is not really necessary, but I can't avoid to display them. I can't leave the title empty, as in this case the name of the device is taken, therefore I just entered '.' as title.

In my view this is the only way to get as much data on the mobile screen. In this case I don't even have to scroll. With all other suggested solutions I end up with the need for scrolling.

![Screenshot_20231202-140340](https://github.com/mennovanhout/flutter-hdashboards-issues/assets/42957854/ef9e9b0a-c731-4e7a-9834-e0eb12ebd79b)

**Comment by mennovanhout** [comment]
I like this idea, but i am pushing this a bit back because this is gonna be a lot of work. But still something I would like to add

**Comment by pilpfiction** [comment]
I know there are a lot other topics, more important. This is more the category "nice to have"...
